spec/controllers/idv/by_mail/enter_code_controller_spec.rb

```diff
@@ -271,7 +271,7 @@
             create(
               :profile,
               :with_pii,
-              fraud_pending_reason: 'threatmetrix_reject',
+              :fraud_rejection,
```

Comment:
Before the user enters their code, they should have a fraud_pending_reason rather than a timestamp set. The timestamp gets added after the code is verified successfully.